finn/finn_def.rb

```diff
@@ -0,0 +1,35 @@
+#practice using functions
+
+def prompt(message)
+	puts "#{message}"
```

Comment:
Please make sure to use 2 space indentation on your code. This follow the normal ruby convention.